infrastructure/rdfdb/cdk/scripts/ebs-mount-script.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+set -e
+set -x
+
+echo "Starting EBS mount script"
+
+# Install AWS CLI
+echo "Installing AWS CLI..."
+curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
```

Comment:
so wearing my security hat, do we need to be checking a sha hash code or anything? Normally we get some security when we use a package manager, but you're just downloading anything from the net. Anyone spoofing a URL could trick this into downloading something else. Can you grab a specific version and hard code the hash code in the script?

Reply:
done